dotnet/samples/Concepts/ChatPrompts/SafeChatPrompts.cs

```diff
@@ -0,0 +1,292 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Text.RegularExpressions;
+using Microsoft.SemanticKernel;
+
+namespace ChatPrompts;
+
+public sealed class SafeChatPrompts : BaseTest
+{
+    private readonly Kernel _kernel;
+
+    public SafeChatPrompts(ITestOutputHelper output) : base(output)
+    {
+        // Create a logging handler to output HTTP requests and responses
+        var handler = new LoggingHandler(new HttpClientHandler(), this.Output);
+        var client = new HttpClient(handler);
```

Comment:
The client needs to be disposed. We can't do this in the constructor because the client will become disposed before any tests will be able to use SK.